framework/src/modules/chain/chain.js

```diff
@@ -564,7 +564,7 @@ module.exports = class Chain {
 	}
 
 	_startLoader() {
-		this.loader.loadTransactionsAndSignatures();
+		this.loader.loadTransactions();
```

Comment:
Would be nice to take the opportunity and give this function a better name like `loadPendingTransactions` or something like that.